pkg/listener/listener.go

```diff
@@ -64,12 +66,26 @@ func (l *StatsDUDPListener) Listen() {
 			level.Error(l.Logger).Log("error", err)
 			return
 		}
-		l.HandlePacket(buf[0:n])
+		// avoid making copies of slices since we need to minimize the time spent here in order not to drop packets
```

Comment:
This comment is confusing to me – `EnqueueUdpPacket` immediately and synchronously goes on to make a copy of the packet?

Reply:
If we really wanted to avoid allocations in the hot path, I guess we would need a rotating pool of buffers, but I'm not sure the memory overhead is worth it, so I would not go there right now. I think the best way here is to delete the confusing comment and accept that we _are_ making a copy.

Reply:
Thank you for taking your time and reviewing my PR. You are right, comment is confusing, I removed the misleading comment. I also tried pooling but I think it made thinks worse since it needs some kind of syncronization.